_posts/2019-10-24-socratic-seminar-97.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+type: socratic
+title: "Socratic Seminar 97"
+meetup: https://www.meetup.com/BitDevsNYC/events/264604145/
+---
+
+Special thanks to Chaincode Labs and NYU Blockchain Digital Asset Forum for food, refreshments and event space.
+
+## Presentation
+
+- Max Galka - [Elementus](https://elementus.io/)
+
+## Mailing Lists and Bitcoin Optech
+
+#### Mailing Lists
+
+##### bitcoin-dev
+- [Taproot updates](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-October/017378.html)
+- [Removal of reject network messages from Bitcoin Core (BIP61)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-October/017400.html)
+
+##### lightning-dev
+
+#### Optech
+
+- [Newsletter #64](https://bitcoinops.org/en/newsletters/2019/09/18/)
+- [Newsletter #65](https://bitcoinops.org/en/newsletters/2019/09/25/)
+- [Newsletter #66](https://bitcoinops.org/en/newsletters/2019/10/02/)
+- [Newsletter #67](https://bitcoinops.org/en/newsletters/2019/10/09/)
+- [Newsletter #68](https://bitcoinops.org/en/newsletters/2019/10/16/)
+
+## Network Data
+
+- [Six Addresses Are Dominating Ethereum (ETH) Usage Right Now](https://bitcoinist.com/six-addresses-are-dominating-ethereum-eth-usage-right-now/)
+- [Today's block statistics](https://bitaps.com/blocks)
+- [Veriblock to use segwit](https://medium.com/@veriblock/veriblock-adopts-segwit-and-will-now-take-more-space-on-bitcoin-edf11fbb6678)
+- [600,000th block](https://twitter.com/pwuille/status/1185346625897517056)
+- [Stale block](https://twitter.com/bitmexresearch/status/1184408265360334849?s=21)
+
+## CVEs and Research
+
+#### Research
+
+- [ABC: Asynchronous Blockchain without Consensus](http://arxiv.org/abs/1909.10926)
+- [Boomerang: Redundancy Improves Latency and Throughput in Payment Networks](http://arxiv.org/abs/1910.01834)
+- [Human readable raw transactions](https://twitter.com/notgrubles/status/1180861313452511233)
+- [Hijacking Routes in Payment Channel Networks: A Predictability Tradeoff](http://arxiv.org/abs/1909.06890)
+- [Privacy-Utility Tradeoffs in Routing Cryptocurrency over Payment Channel Networks](http://arxiv.org/abs/1909.02717)
+- [Short Paper: Towards Characterizing Sybil Attacks in Cryptocurrency Mixers](https://eprint.iacr.org/2019/1111)
+- [Simplicity's first transaction](https://lists.ozlabs.org/pipermail/simplicity/2019/000018.html)
+- [Transparent SNARKs from DARK Compilers](https://eprint.iacr.org/2019/1229.pdf)
+- [Binary quadtratic forms](http://diyhpl.us/%7Ebryan/papers2/bitcoin/vdf-competition-classgroups-binary-quadratic-form.pdf)
+- [Bitcoin Chain Width ExpansionDenial-of-Service Attacks](https://bcoin.io/papers/bitcoin-chain-expansion.pdf)
+
+
+#### InfoSec
+
+- [lnd, c-lightning, eclair CVE-2019-12998 / CVE-2019-12999 /	CVE-2019-13000](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-September/002174.html) [Rusty broadcast](https://twitter.com/i/broadcasts/1yqkdnryandjv)
```

Comment:
It's difficult to see that this is two links. I suggest making the second link a sub-bullet.

Reply:
Also, rusty broadcast is broken.

Reply:
Changed to direct links to the tweets, the broadcast links appear unreliable.